### PR TITLE
Add clappr-playback-name-plugin version 0.0.1

### PR DESCRIPTION
**Pull request by barbosa**

### CODE REVIEW

**Comment by jimaek** [comment]
Can you rename the directory to `clappr.playback-name` ?

**Comment by jimaek** [comment]
Also no reason to add the unminified file. Just the min is enough

**Comment by barbosa** [comment]
Done!

File: files/clappr.playback-name/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "clappr-playback-name-plugin",
+  "repo": "barbosa/clappr-playback-name-plugin",
+  "files": {
+    "basePath": "dist/",
+    "include": ["playbackname.js", "playbackname.min.js"]
```
Comment (jimaek) [inline files/clappr.playback-name/update.json]: 1. Remove the unminified file from here too
2. There is not dist/ directory in your repo.

**Comment by jimaek** [comment]
Once you fix all the issues please squash your commits into 1. Thanks

**Comment by barbosa** [comment]
Hi @jimaek 

Just fixed the issues and squashed the 3 commits into 1. Thanks.

**Comment by jimaek** [comment]
Thank you

**Pull request merged**